e2e-identity/tests/utils/docker/wiremock.rs

```diff
@@ -46,26 +44,28 @@ impl WiremockImage {
 }
 
 impl Image for WiremockImage {
-    type Args = WiremockArgs;
-
-    fn name(&self) -> String {
-        Self::NAME.to_string()
+    fn name(&self) -> &str {
+        Self::NAME
     }
 
-    fn tag(&self) -> String {
-        Self::TAG.to_string()
+    fn tag(&self) -> &str {
+        Self::TAG
     }
 
     fn ready_conditions(&self) -> Vec<WaitFor> {
         vec![WaitFor::seconds(1)]
     }
 
-    fn volumes(&self) -> Box<dyn Iterator<Item = (&String, &String)> + '_> {
-        Box::new(self.volumes.iter())
+    fn mounts(&self) -> impl IntoIterator<Item = &Mount> {
+        &self.volumes
+    }
+
+    fn cmd(&self) -> impl IntoIterator<Item = impl Into<Cow<'_, str>>> {
+        vec!["/stubs".to_string(), "-p".to_string(), WiremockImage::PORT.to_string()].into_iter()
```

Comment:
Two questions here:
- what's `/stubs` and what does `/stubs -p` do?
- why call `into_iter` when `Vec` impls `IntoIterator`?